[
    {
        "type": "error",
        "module": "preprocessing_test",
        "obj": "",
        "line": 6,
        "column": 0,
        "endLine": 6,
        "endColumn": 49,
        "path": "src/test/preprocessing_test.py",
        "symbol": "import-error",
        "message": "Unable to import 'preprocessing'",
        "message-id": "E0401"
    },
    {
        "type": "convention",
        "module": "preprocessing_test",
        "obj": "",
        "line": 6,
        "column": 0,
        "endLine": 6,
        "endColumn": 49,
        "path": "src/test/preprocessing_test.py",
        "symbol": "wrong-import-position",
        "message": "Import \"from preprocessing import Preprocessing, final_df\" should be placed at the top of the module",
        "message-id": "C0413"
    },
    {
        "type": "warning",
        "module": "preprocessing_test",
        "obj": "test_scalarization",
        "line": 13,
        "column": 23,
        "endLine": 13,
        "endColumn": 32,
        "path": "src/test/preprocessing_test.py",
        "symbol": "redefined-outer-name",
        "message": "Redefining name 'dataframe' from outer scope (line 9)",
        "message-id": "W0621"
    },
    {
        "type": "warning",
        "module": "preprocessing_test",
        "obj": "",
        "line": 4,
        "column": 0,
        "endLine": 4,
        "endColumn": 19,
        "path": "src/test/preprocessing_test.py",
        "symbol": "unused-import",
        "message": "Unused pandas imported as pd",
        "message-id": "W0611"
    }
]
